Trimble.Modus.Components/Controls/TMInput/CustomInput.xaml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Trimble.Modus.Components.Controls.CustomInput"
+             x:Name="tmInput"
+             xmlns:converters="clr-namespace:Trimble.Modus.Components.Converters"
+             xmlns:controls="clr-namespace:Trimble.Modus.Components">
+
+  <ContentView.Resources>

```

Comment:
And the password eye icon also not working now. Previously it used to show/hide the text in the input field

Reply:
Changed and resolved the issue